promitor-agent-scraper/values.yaml

```diff
@@ -175,9 +177,28 @@ rbac:
     name: promitor-scraper
     annotations: {}
 
+    ## Set this to true if you plan on using Pod Security Policy
+    automountServiceAccountToken: false
+
 nodeSelector: {}
 
+## securityContext and containerSecurityContext are using secure defaults.
+## Only override if you have a good reason to.
 securityContext:
-  enabled: false
+  enabled: true
```

Comment:
So if we turn this on and I don't do anything special, this will still pass - Correct?

Reply:
Yes. Running as user 10000 now works fine because we changed the port to 5000, and I have yet to see a single pod that does not work correctly with RuntimeDefault seccompProfile.